main.go

```diff
@@ -32,67 +32,95 @@ import (
 	"github.com/sapcc/go-bits/logg"
 	"github.com/sapcc/go-bits/must"
+	flag "github.com/spf13/pflag"
 	"go.uber.org/automaxprocs/maxprocs"
 
 	"github.com/sapcc/swift-health-exporter/internal/collector"
 	"github.com/sapcc/swift-health-exporter/internal/collector/dispersion"
 	"github.com/sapcc/swift-health-exporter/internal/collector/recon"
 )
 
-var cli struct {
-	Debug            bool   `env:"DEBUG" help:"Enable debug mode."`
-	ShowVersion      bool   `name:"version" short:"v" help:"Report version string and exit."`
-	WebListenAddress string `name:"web.listen-address" default:"0.0.0.0:9520" help:"Exporter listening address."`
+var (
+	debug            bool
+	showVersion      bool
+	webListenAddress string
 
-	MaxFailures int `name:"collector.max-failures" default:"4" help:"Max allowed failures for a specific collector."`
+	maxFailures int
 
 	// In large Swift clusters the dispersion-report tool takes time, therefore we have a higher default timeout value.
-	DispersionTimeout   int64 `name:"dispersion.timeout" default:"20" help:"Timeout value (in seconds) for the context that is used while executing the swift-dispersion-report command."`
-	DispersionCollector bool  `name:"collector.dispersion" help:"Enable dispersion collector."`
-
-	ReconTimeout                   int64 `name:"recon.timeout" default:"4" help:"Timeout value (in seconds) for the context that is used while executing the swift-recon command."`
-	ReconHostTimeout               int   `name:"recon.timeout-host" default:"1" help:"Timeout value (in seconds) that is used for the '--timeout' flag (host timeout) of the swift-recon command."`
-	NoReconMD5Collector            bool  `name:"no-collector.recon.md5" help:"Disable MD5 collector."`
-	ReconDiskUsageCollector        bool  `name:"collector.recon.diskusage" help:"Enable disk usage collector."`
-	ReconDriveAuditCollector       bool  `name:"collector.recon.driveaudit" help:"Enable drive audit collector."`
-	ReconQuarantinedCollector      bool  `name:"collector.recon.quarantined" help:"Enable quarantined collector."`
-	ReconReplicationCollector      bool  `name:"collector.recon.replication" help:"Enable replication collector."`
-	ReconShardingCollector         bool  `name:"collector.recon.sharding" help:"Enable sharding collector."`
-	ReconUnmountedCollector        bool  `name:"collector.recon.unmounted" help:"Enable unmounted collector."`
-	ReconUpdaterSweepTimeCollector bool  `name:"collector.recon.updater_sweep_time" help:"Enable updater sweep time collector."`
+	dispersionTimeout   int64
+	dispersionCollector bool
+
+	reconTimeout                   int64
+	reconHostTimeout               int
+	noReconMD5Collector            bool
+	reconDiskUsageCollector        bool
+	reconDriveAuditCollector       bool
+	reconQuarantinedCollector      bool
+	reconReplicationCollector      bool
+	reconShardingCollector         bool
+	reconUnmountedCollector        bool
+	reconUpdaterSweepTimeCollector bool
+)
+
+func init() {
```

Comment:
I think we want to move the code from the init functions into the beginning of main as we generally try to avoid using init() as much as possible due to its surprises when importing libraries.